src/main/java/org/folio/list/rest/ConfigurationClient.java

```diff
@@ -1,12 +1,39 @@
 package org.folio.list.rest;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import feign.FeignException;
+import java.time.DateTimeException;
+import java.time.ZoneId;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.stereotype.Service;
 import org.springframework.web.bind.annotation.GetMapping;
 
+@Log4j2
 @Service
 @FeignClient(name = "configurations")
-public interface ConfigurationClient {
+@RequiredArgsConstructor(onConstructor_ = @Autowired)
+public abstract class ConfigurationClient {
+
+  private final ObjectMapper objectMapper;
+
   @GetMapping("/entries?query=(module==ORG and configName==localeSettings)")
-  String getLocaleSettings();
+  public abstract String getLocaleSettings();
+
+  public ZoneId getTenantTimezone() {
```

Comment:
Moved the logic for this to here, to decrease unrelated complexity in the already nasty `UserFriendlyQueryService`.